### PR TITLE
Adding a simple UDF

### PR DESCRIPTION
**Pull request by appan-altiscale**

### CODE REVIEW

File: hive-examples/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <configuration>
+                <archive>
+                    <manifest>
+                        <mainClass>com.altiscale.hive.udf.ToLower</mainClass>
+                    </manifest>
+                </archive>
+                <descriptorRefs>
+                    <descriptorRef>jar-with-dependencies</descriptorRef>
```
Comment (bala-altiscale) [inline hive-examples/pom.xml]: Can we suggest to go with non-fat jars?

File: hive-examples/pom.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          </releases>
+          <snapshots>
+          <enabled>false</enabled>
+          <updatePolicy>never</updatePolicy>
+          <checksumPolicy>fail</checksumPolicy>
+          </snapshots>
+          <id>HDPReleases</id>
+          <name>HDP Releases</name>
+          <url>http://repo.maven.apache.org/</url> 
+          <layout>default</layout>
```
Comment (bala-altiscale) [inline hive-examples/pom.xml]: What is the significance of "HDPReleases" here?

File: hive-examples/pom.xml (hunk 1 above)
Comment (appan-altiscale) [inline hive-examples/pom.xml]: It is not building the fat jar. Is there any specific change you want to make? Please let me know.

File: hive-examples/pom.xml (hunk 2 above)
Comment (appan-altiscale) [inline hive-examples/pom.xml]: Fixed.

**Comment by bala-altiscale** [comment]
LGTM. merged.

**Pull request merged**